chapters/1-rationale.md

```diff
@@ -90,7 +96,7 @@ In an SPDX document, Relationship elements can be used to indicate relationships
 
 ## 1.8 Conformance <a name="1.8"></a>
 
-**1.8.1** A file can be designated an SPDX document, if it is compliant with the requirements of the SPDX Trademark License (See the SPDX Trademark Page).
+**1.8.1** A file can be designated an SPDX document, if it is compliant with the requirements of the SPDX Trademark License (See the SPDX Trademark Page on the spdx.org web site).
 
 **1.8.2** The official copyright notice to be used with any verbatim reproduction and/or distribution of this SPDX Specification 2.2-DRAFT is:
```

Comment:
Should "-DRAFT" be taken out now? Or are we separately tracking this elsewhere to make sure that it's removed before the new version is released?